kedro/config/config.py

```diff
@@ -51,8 +51,10 @@ class ConfigLoader(AbstractConfigLoader):
 
         >>> import logging.config
         >>> from kedro.config import ConfigLoader
+        >>> from kedro.framework.project import settings
         >>>
-        >>> conf_loader = ConfigLoader('conf', 'local')
+        >>> conf_path = str(project_path / settings.CONF_SOURCE)
```

Comment:
Thank you for updating this!

Is `project_path` something available in settings already? Can we make `conf_path` available in settings so we can import it instead of asking users to compose the path themselves?

Reply:
No it's not. I was deciding between `project_path` or `Path.cwd()`, and chose the former because I thought it would be clearer to the user what path they should be providing. The user would only ever need to construct this path themselves if they want to create a custom `ConfigLoader` or access the object directly in a notebook as far as I'm aware. So IMO it's fine to keep it like this because they won't usually need to construct anything at all.

Reply:
That makes sense, thank you.